src/dguweb/web/templates/user/index.html.eex

```diff
@@ -10,10 +10,9 @@
         <% end %>
 
         <ul>
-        <%= for {pubname, pubmap} <- @publishers do %>
+        <%= for {name, title} <- @publishers do %>
             <li>
-            <%= link pubmap.title, to: publisher_path(@conn, :show, pubname) %>
-            (<%= pubmap.role %>)
+            <%= link title, to: publisher_path(@conn, :show, name) %>
```

Comment:
Until now I've been indenting template markup with HTML tags. I'm happy either way, but it doesn't hurt to be consistent.